sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java

```diff
@@ -657,6 +657,11 @@ static void assertRestException(HttpResponseException exception, int expectedSta
         assertRestException(exception, HttpResponseException.class, expectedStatusCode);
     }
 
+    static void assertRestException(Throwable exception, Class<? extends HttpResponseException> expectedExceptionType, int expectedStatusCode) {
```

Comment:
nit: call this `assertResponseException` as the exception is of type `HttpResponseException`.